src/components/ConfirmationBox/ConfirmationBox.test.js

```diff
@@ -5,32 +5,38 @@ import ConfirmationBox from './ConfirmationBox';
 
 configure({adapter: new Adapter()});
 
-it('calls the onYes correct functions when clicked', () => {
-	const mockFuncYes = jest.fn();
-	const mockFuncNo = jest.fn();
-	const wrapper = shallow(
-		<ConfirmationBox
-			item={{name: 'test', id: 'xxx'}}
-			onYes={mockFuncYes}
-			onNo={mockFuncNo}
-			store={{}}
-		/>
-	);
-	wrapper.find('[data-test-id="YES"]').simulate('click');
-	expect(mockFuncYes).toHaveBeenCalledTimes(1);
+const getProps = () => {
+	return {
+		setActualItem: jest.fn(),
+		storeList: {xxx: {id: 'xxx', name: 'Alfred'}},
```

Comment:
Small point but your mocked constants should be either as realistic or as generic as possible. This passes but really it should be either an actual item, or something like ```foobar```

also is the id generated by the model a string? if so fine, if not then please mimic it properly